doc/association-rules.rst

```diff
@@ -0,0 +1,79 @@
+Association Rules
+=================
+
+.. figure:: icons/association-rules.png
+
+Induction of association rules.
+
+Signals
+-------
+
+#### Inputs
+
+-  **Data**
+
+Data set
+
+#### Outputs
+
+-  **Matching Data**
+
+Data instances matching the criteria.
+
+Description
+-----------
+
+This widget runs the original `Agrawal's algorithm <http://cs-people.bu.edu/evimaria/cs565/advances.pdf>`__ for the induction of
+association rules. For inducing classification rules, it generates rules
+for the entire itemset and filters out the rules where the consequent does 
+not match one of the class' values.
+
+.. figure:: images/association-rules-stamped.png
+
+1. Information on the data set.
+
+2. In *Find association rules* you can set criteria for rule induction:
+
+   -  **Minimal support**: percentage of the entire data set covered by the entire rule (antecedent and consequent).
+   -  **Minimal confidence**: proportion of the number of examples which fit the right side (consequent) among those that fit the left side (antecedent).
+   -  **Max. number of rules**: limit to the number of rules the algorithm generates. Too many rules can slow down the widget considerably.
+
+   If *Induce classification (itemset → class) rules* is ticked, the widget
+   will only generate rules that have a class value on the right-hand side
+   (consequent) of the rule.
+
+   If *Auto find rules is on*, the widget will run the search at every change of
+   parameters. Might be slow for data sets with many attributes, so pressing *Find 
+   rules* only when the parameters are set is a good idea.
+
+3. *Filter rules* by:
+
+   -  **Antecedent**:
+         - *Contains*: will filter rules by matching space-separated regular expressions https://en.wikipedia.org/wiki/Regular_expression in antecedent items.
```

Comment:
`... space-separated [regular expressions] in antecedent items.`

Then at the end of this paragraph:
```markdown
[regular expressions]: https://en.wikipedia.org/wiki/Regular_expression
```